projects/network_tester/src/ui_task.c

```diff
@@ -657,6 +673,35 @@ static void ui_load_menu(void)
             screen[(menu_pos%3)+2][0] = CURSOR_GLYPH;
             xTaskNotifyGive(s_screen_task_handle);
             break;
+        case SET_DL_BAND_MENU:
+            menu_pos_floor = (menu_pos/3) * 3;
+            llabs_dl_band_cfg_t dl_band_cfg;
+            int8_t active_pos;
+            if (ll_dl_band_cfg_get(&dl_band_cfg) != 0) {
+                if (dl_band_cfg.band_edge_lower == FCC.band_edge_lower)              active_pos = 0;
```

Comment:
C doesn't let us use a switch here :-1: @awelkiell 

`error: case label does not reduce to an integer constant`

http://stackoverflow.com/questions/24822887/use-const-struct-member-in-switch-case-statement